pdf_generator.py

```diff
@@ -0,0 +1,171 @@
+import fpdf
+import copy
+class PdfGenerator:
+    def __init__(self, invoice):
+        self.invoice = invoice
+        self.pdf = fpdf.FPDF(orientation='P', unit='mm', format='A4')
+        self.pdf.add_page()
+        self.pdf.set_font("Arial")
+
+        self.initialX = self.pdf.get_x()
+        self.initialY = self.pdf.get_y()
+
+        self.tableColumnDateWidth = 25
+        self.tableColumnClientCodeWidth = 35
+        self.tableColumnDescriptionWidth = 80
+        self.tableColumnHoursWidth = 10
+
+    def generatePdf(self):
+        self.create_seller_info()
```

Comment:
I appreciate that you grouped related rendering code into a single function with a clear name.
We'll take this process to the next level together.